src/Language/Fortran/Util/Position.hs

```diff
@@ -26,7 +26,7 @@ instance Binary Position
 instance NFData Position
 
 instance Show Position where
-  show (Position _ c l _ _) = show l ++ ':' : show c
+  show (Position _ c l _ _) = show l ++ ':' : show (c - 1)
```

Comment:
Could you add your 2nd point in the PR and the PR number #292 here as a comment?